frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/DeleteDataset/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button as RawButton, H6, Intent } from "@blueprintjs/core";
```

Comment:
Extract Seve's Delete button and Alert modal to be a component, so it's reused in both trashcan and inline cancel button